pandas/tests/io/excel/test_readers.py

```diff
@@ -19,16 +19,27 @@
 
 
 @contextlib.contextmanager
-def ignore_xlrd_time_clock_warning():
+def ignore_expected_engine_warnings():
```

Comment:
@tdamsma I see you put this back but I intentionally removed as it wasn't causing issues on my end (and shouldn't be in one context manager anyway)

The comment you have linked shows Python 2.7 - is that what you are using to run this? We don't support that any more if so

Reply:
I think the issue has resolved itself by switching to read_only mode openpyxl doesn't throw these warnings anymore. I dropped the commit.

Reply:
Oops, so I kind of messed up accidentally removed the branch on origin.